lms/templates/dashboard/_dashboard_entitlement_actions.html

```diff
@@ -43,6 +43,19 @@
           </a>
         </li>
       % endif
-    </ul>
+      % if show_email_settings and entitlement_session:
+        <li class="entitlement-actions-item" id="actions-item-email-settings-${dashboard_index}" role="menuitem">
+          ## href, id, rel, and data-model-close-button-selector must be defined for compatibility with lms/static/js/leanModal.js
```

Comment:
If you're relying on this being picked up by legacy.js, ID is not required here.